add dz-4

bmusienko opened the pull request.

bmusienko committed:
add dz-4

Voronve commented on 37-musienko-bohdan/dz-4/css/index.css:
```diff
@@ -0,0 +1,50 @@
+	background: #39b54a;
+}
+.silver {
+	background: #eae1ce;	
```
Класси мають бути названі більш прив'язано до того на що вони вказують. Не просто колір а назва елемента наприклад.

Voronve commented on 37-musienko-bohdan/dz-4/css/index.css:
```diff
@@ -0,0 +1,50 @@
+}
+.list-legend li:nth-child(4) .block {
+	background: #838383;
+}
```
Замість того, щоб знову вказувати ті ж кольори напряму можна просто присвоїти елементам ті самі класи що означають кольори

bmusienko committed:
add dz-4

bmusienko committed:
merge
Merge branch 'feature/37-bohdan-musienko/dz-4' of https://github.com/Evan-Kork/temabit-lections-2019 into feature/37-bohdan-musienko/dz-4

Evan-Kork reviewed (commented):
Чекаю правок по коментарях

bmusienko committed:
change name classes

bmusienko commented:
все поправив

Evan-Kork commented on 37-musienko-bohdan/dz-4/css/index.css:
```diff
@@ -0,0 +1,44 @@
+.empty {
+	background: #eae1ce;
+}
+tr:nth-child(4) {
```
Без nth-child реалізувати

bmusienko committed:
change nth-child

bmusienko commented:
виправи

bmusienko commented:
dz-4 done
